metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -363,4 +456,8 @@ protected EntityRegistry getEntityRegistry() {
   }
 
   public abstract void setWritable(boolean canWrite);
+
+  public abstract EbeanEntityService.RollbackRunResult rollbackRun(List<AspectRowSummary> aspectRows, String runId);
```

Comment:
RollbackrunResult should not be tied to a subclass of this abstract class

Reply:
good spot- will update